### PR TITLE
Add news post for playtest-20150531

### PR DESCRIPTION
**Pull request by obrakmann**

### CODE REVIEW

**Comment by obrakmann** [comment]
Updated. TD's maps are all very small, so they all show the blocky minimap, so I had to switch to a ra screenshot (which doesn't show the zoom and grid buttons then, unfortunately).

Also added the playtest id now.

**Comment by Mailaender** [comment]
Updated https://build.opensuse.org/package/show/games:openra/playtest

File: content/news/playtest-20150531.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+We have received a small number of issues after last week's playtest release. These issues have been addressed and so we are releasing a new playtest for you to check out!
+
+The list of fixes is agreeably short, which is to be expected after this equally short development cycle. The fixed issues include:
+
+* A number of issues pertaining to single-player missions in both Tiberian Dawn and Red Alert.
+* A crash in Dune 2000 that happened when a Deviator hit a harvester being picked up by a Carryall.
+* Rendering glitches which resulted in highly saturated range circles and blocky nuke explosion effects.
+
+For the full changelog, please refer to http://changelog.openra.net.
```
Comment (Mailaender) [inline content/news/playtest-20150531.md]: Is this not linked on purpose?

File: content/news/playtest-20150531.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+The installers for a number of supported operating systems can be found on [the download page](/download/) as usual.
+
+If you encounter any problems or bugs, don't hesitate to report them [on our bug tracker](http//bugs.openra.net).
```
Comment (Mailaender) [inline content/news/playtest-20150531.md]: > If you encounter any problems...

I am not sure if we should advertise the bug tracker as a support ticket system for general questions even though our forums suck a lot.

Comment (Mailaender) [inline content/news/playtest-20150531.md]: Also the link `http//bugs.openra.net` is broken.

**Comment by obrakmann** [comment]
Updated with fixed links.

I've always used that language when linking to the bug tracker in past announcements.

**Comment by Mailaender** [comment]
Thanks!

**Pull request merged**